examples/pxScene2d/src/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ option(BUILD_PXSCENE_STATIC_LIB "BUILD_PXSCENE_STATIC_LIB" OFF)
 option(BUILD_PXWAYLAND_SHARED_LIB "BUILD_PXWAYLAND_SHARED_LIB" OFF)
 option(BUILD_PXWAYLAND_STATIC_LIB "BUILD_PXWAYLAND_STATIC_LIB" ON)
 option(BUILD_PXSCENE_WITH_NEXUS_SUPPORT "BUILD_PXSCENE_WITH_NEXUS_SUPPORT" OFF)
+option(ENABLE_DEBUG_METRICS "ENABLE_DEBUG_METRICS" OFF)
```

Comment:
Please rename option from "ENABLE_DEBUG_METRICS" to "BUILD_DEBUG_METRICS".  The add the following block of code after this option:

if (BUILD_DEBUG_METRICS)
  message("Building with debug metrics")
  add_definitions(-DENABLE_DEBUG_METRICS)
endif (BUILD_DEBUG_METRICS)